stdeb/util.py

```diff
@@ -1413,18 +1413,28 @@ def build_dsc(debinfo,
                          'compatible with older versions of debhelper.'%(
                     DH_MIN_VERS,))
 
-        python_defaults_version_str = get_version_str('python-all')
-        if len(python_defaults_version_str)==0:
-            log.warn('This version of stdeb requires python-all >= %s, '
-                     'but you do not have this package installed. '
-                     'Could not check compatibility.'%PYTHON_ALL_MIN_VERS)
-        else:
-            if not dpkg_compare_versions(
-                python_defaults_version_str, 'ge', PYTHON_ALL_MIN_VERS):
-                log.warn('This version of stdeb requires python-all >= %s. '
-                         'Use stdeb 0.6.0 or older to generate source packages '
-                         'that use python-support.'%(
-                    PYTHON_ALL_MIN_VERS,))
+        if debinfo.with_python2:
+            python_defaults_version_str = get_version_str('python-all')
+
+            if len(python_defaults_version_str) == 0:
+                log.warn('This version of stdeb requires python-all >= %s, '
+                         'but you do not have this package installed. '
+                         'Could not check compatibility.' % PYTHON_ALL_MIN_VERS)
+            else:
+                if not dpkg_compare_versions(
+                        python_defaults_version_str, 'ge', PYTHON_ALL_MIN_VERS):
+                    log.warn('This version of stdeb requires python-all >= %s. '
+                             'Use stdeb 0.6.0 or older to generate source packages '
+                             'that use python-support.' % (
+                                 PYTHON_ALL_MIN_VERS,))
+
+        if debinfo.with_python3:
+            python3_defaults_version_str = get_version_str('python3-all')
+
+            if len(python3_defaults_version_str) == 0:
+                log.warn('This version of stdeb requires python3-all >= %s, '
+                         'but you do not have this package installed. '
+                         'Could not check compatibility.' % PYTHON_ALL_MIN_VERS)
```

Comment:
Showing the Python 2 minimum version here seem wrong?

Reply:
@Sispheor Friendly ping.

Reply:
What do you mean?

Reply:
The current [setup](https://github.com/astraw/stdeb/blob/master/setup.py#L30) point to python 3 with no specific minor version.
The var PYTHON_ALL_MIN_VERS is set to '2.6.6-3'.
You want me to add a new var called PYTHON3_ALL_MIN_VERS = 3 ?

Reply:
The previous message said something like "You need python3-all > 2.6.x" which doesn't make any sense.

What motivates to pick a specific `PYTHON3_ALL_MIN_VERS`? Is there any specific constrain or can the message just refer to `python3-all` without a minimum version?

Reply:
@Sispheor Friendly ping.

Reply:
In order to move forward on this I have removed the `PYTHON3_ALL_MIN_VERS` in 05734f61f6ca1e0ba1bf0844a16daf7ec2740413.